lib/KBreadcrumbs.vue

```diff
@@ -266,8 +270,13 @@
   }
 
   .breadcrumbs-dropdown {
+    @extend %dropshadow-16dp;
+
+    position: absolute;
+    z-index: 100;
     padding: 16px;
     font-weight: bold;
+    background: white;
```

Comment:
technically, this should be set using the `surface` token in the theme:

https://kolibri-design-system.netlify.com/colors/#tokens-surface